flake.nix

```diff
@@ -198,6 +199,11 @@
         devShell = pkgs.mkShell {
           inherit CARGO_TARGET_DIR;
           buildInputs = [ fenixStable ] ++ buildDeps;
+          shellHook = ''
+            export ROOT_DIR=$(dirname "$(realpath ./flake.nix)")
+            export AZTEC_SRS_PATH="$ROOT_DIR/data/aztec20/kzg10-aztec20-srs-1048584.bin"
```

Comment:
Is this file always guaranteed to exist?

Reply:
no, the following line `./download_srs_aztec.sh` will download it if not exists, and skip if already exist, and this check is done every time we enter the dev shell.